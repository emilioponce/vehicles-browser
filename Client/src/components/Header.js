import React from 'react'

import logo from '../images/logo.svg'
import styles from './Header.module.css'

const Header = () => {
  return (
    <div className={styles.wrapper}>
      <div className={styles.header}>
        <img src={logo} className={styles.logo} alt="Vehicles Browser" />
        <div className={styles.title}>Vehicles Browser</div>
        <div className={styles.subtitle}>
          A tool for visualizing different types of vehicles
        </div>
      </div>
    </div>
  )
}

export default Header
