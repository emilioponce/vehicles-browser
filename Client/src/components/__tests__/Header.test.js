import React from 'react'
import ReactDOM from 'react-dom'
import Header from '../Header'
import renderer from 'react-test-renderer'

const component = <Header />
describe('Component Header', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(component, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('snapshot renders correctly', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
