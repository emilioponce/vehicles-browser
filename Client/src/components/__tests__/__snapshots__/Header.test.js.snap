// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header snapshot renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="header"
  >
    <img
      alt="Vehicles Browser"
      className="logo"
      src="logo.svg"
    />
    <div
      className="title"
    >
      Vehicles Browser
    </div>
    <div
      className="subtitle"
    >
      A tool for visualizing different types of vehicles
    </div>
  </div>
</div>
`;
